src/Cohesity.Powershell/Scripts/Utility/Invoke-RestApi.ps1

```diff
@@ -117,6 +117,17 @@ function Invoke-RestApi {
     catch {
         # this flag can be optionally used by the caller to identify the details of failure
         $Global:CohesityAPIError = $_.Exception
+        try  {
+           $Global:CohesityAPIError | Test-Json
+        }
+        catch  {
+
+            $errorMsg = "Invalid JSON format. Unable to parse the JSON string..."
```

Comment:
The error message could be better. Developer needs to understand what the error is, Can you print the response ? 
"Unable to read incoming request from server" : $Response

Also test these scenarios:
1.  API which returns Error
2. Try API which is not in the system
3.  the powershell lint is failing on unrelated error, can you fix that ?